xcm/src/v3/multiasset.rs

```diff
@@ -107,6 +109,130 @@ impl From<[u8; 32]> for AssetInstance {
 	}
 }
 
+impl From<u8> for AssetInstance {
+	fn from(x: u8) -> Self {
+		Self::Index(x as u128)
+	}
+}
+
+impl From<u16> for AssetInstance {
+	fn from(x: u16) -> Self {
+		Self::Index(x as u128)
+	}
+}
+
+impl From<u32> for AssetInstance {
+	fn from(x: u32) -> Self {
+		Self::Index(x as u128)
+	}
+}
+
+impl From<u64> for AssetInstance {
+	fn from(x: u64) -> Self {
+		Self::Index(x as u128)
+	}
+}
```

Comment:
Missing `From<u128>` implementation for `AssetInstance`?

Reply:
Oh I see now, `From<u128>` is deliberately not implemented because that's reserved for fungibles.